persistence/startup-folder/write-file-to-startup-folder.yml

```diff
@@ -12,9 +13,17 @@ rule:
     examples:
       - 07F7846BBCDA782E5639292AD93907EB:0x401040
   features:
-    - and:
-      - match: get startup folder
-      - or:
-        - match: copy file
-        - match: move file
-        - match: host-interaction/file-system/write
+    - or:
+      - and:
+        - match: get startup folder
+        - or:
+          - match: copy file
+          - match: move file
+          - match: write file on Windows
+      - call:
+        - and:
+          - match: reference startup folder
```

Comment:
`reference startup folder` has file-level scope so this doesn't work, for this instance I'd say we copy the string in here directly.